source/state/vault/index.ts

```diff
@@ -308,13 +308,23 @@ const VaultState = createSlice({
     },
     setFaucetModalState: (
       state: IVaultState,
-      action: PayloadAction<number>
+      action: PayloadAction<{ chainId: number; isFirstTime?: boolean }>
```

Comment:
with the migration isFirstTime is not needed anymore?

Reply:
are separate cases, isfirstime will still remain necessary as we need to know if the user has not yet interacted with the modals in the faucet networks